net.eudic.dict.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: net.eudic.dict
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+rename-icon: eudic
+rename-desktop-file: eudic.desktop
+command: eudic 
+finish-args:
+  - --share=ipc
+  - --share=network
```

Comment:
why does it need network?

Reply:
User can login to sync, or download dictionaries from server.